kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/DataViewModelHelper.java

```diff
@@ -41,4 +47,12 @@ public class DataViewModelHelper {
             INSTANT_LITERAL,
             LONG
     ));
+
+    public static Set<ConceptEntity> fetchDescendentsOfConcept(ViewProperties viewProperties, ConceptFacade conceptFacade) {
```

Comment:
Can the `conceptFacade` parameter be just the `PublicId` and not a `ConceptFacade`?